isis/src/base/objs/LidarData/LidarData.cpp

```diff
@@ -344,10 +361,19 @@ namespace Isis {
    * @throws IException::User Throws User exception if it cannot open the file for writing.
    */
   void LidarData::write(FileName outputFile, LidarData::Format format) {
+    bool sort = false;
```

Comment:
Do you need this bool? I don't see it used anywhere but in "points(sort). Can you just use the "false" directly and put a comment explaining the this means the points are not being serialized in any particular order?